README.md

```diff
@@ -11,27 +11,29 @@ The Action can optionally wait for analysis completion, writing the final status
 The workflow will be set to fail if:
 
 - The source/binaries glob(s) fail to match any files
-- There are any errors when contacting your Code Dx server
+- There are any errors when contacting your SRM server
 - The analysis ends in failure
 
 ## Requirements
 
-- A deployed, licensed instance of Code Dx (any license)
-- Access from GitHub to Code Dx via HTTP, or via HTTPS with a recognizable certificate (use the `ca-cert` param if not using a public CA)
-- A Project in Code Dx to store results
+- A deployed, licensed instance of SRM (any license)
+- Access from GitHub to SRM via HTTP, or via HTTPS with a recognizable certificate (use the `ca-cert` param if not using a public CA)
+- A Project in SRM to store results
 - An API Key or Personal Access Token with "Create" permissions for the Project
 
 ## Action Inputs
 
 | Input Name                 | Description                                                                                              | Default Value | Required |
 |----------------------------|----------------------------------------------------------------------------------------------------------|---------------|----------|
-| `server-url`               | The URL for the Code Dx server (typically ends with `/codedx`)                                           |               | Yes      |
-| `api-key`                  | An API Key or Personal Access Token to use when connecting to Code Dx                                    |               | Yes      |
-| `project-id`               | The ID of a project (an integer) created in Code Dx                                                      |               | Yes      |
-| `source-and-binaries-glob` | A comma-separated-list of file globs matching source and binary files to be packaged and sent to Code Dx | `undefined`   | No       |
+| `server-url`               | The URL for the SRM server (typically ends with `/codedx`)                                           |               | Yes      |
+| `api-key`                  | An API Key or Personal Access Token to use when connecting to SRM                                    |               | Yes      |
+| `project-id`               | The ID of a project (an integer) created in SRM                                                      |               | Yes      |
+| `base-branch-name`         | The parent branch name of a project created in SRM                                                   | `undefined`   | No       |
+| `target-branch-name`       | The target branch name of a project created in SRM                                                   | `undefined`   | No       |
```

Comment:
The `base-branch-name` is _technically_ required if `target-branch-name` is specified and doesn't exist yet, but the "Required" column doesn't have enough space to give that distinction

I suggest changing to `No*` for `base-branch-name` and add a note below this table mentioning this